views/partials/subpages/confirm.hbs

```diff
@@ -0,0 +1,3 @@
+<span class="dashboard-email-sent bold"> {{ getString "email-verified" }}</span>
+<h1>{{ getString "email-added-to-subscription" email=email }}</h1>
+{{{ getString "email-verified-view-dashboard" sign-in-link='<a href="/oauth/init">Sign in</a>'}}}
```

Comment:
This doesn't look right? You seem to be hard-coding text.

Reply:
Yup, noted in https://github.com/mozilla/blurts-server/pull/1196#pullrequestreview-292592329. @lesleyjanenorton is going to help fix this with her expertise.